NEWS.md

```diff
@@ -3,6 +3,7 @@
 ### Bug Fixes
 
 - Previously, the compound symmetry correlation models (`cs` and `csh`) could have divergence issues, especially when the correlation was negative. This was because the range of the correlation parameter was from -1 to 1, whereas the correct range is between -1/(m-1) and 1, where m is the number of observations per subject. This is fixed now by using another mapping from the real line to the correct range, and therefore compound symmetry correlation models will work more robustly. 
+- Previously, the `emp_start` starting values could only work with variables included in `data`. Now they can also work when variables are not included in `data`.
```

Comment:
it sounds a bit strange, that the issue itself describes that the prediction is not working if response is not included, indeed this issue arise from the fact that we want to build the tmb_data to ensure everything is consistent (including the factor levels etc). however why the issue goes to `emp_start`? because `emp_start` use a linear regression and then use the residuals to obtain the covariance matrix shape. It is indeed strange that it is still working when the response is not available, how would we provide this "empirical start values"?

Reply:
by the way, in terms of "variables" included in the `data`, does it mean

1. response is `log(y)` however data contains only `y`
2. response is `y` however data does not contain any `y`
3. independent variable include `x` while data does not contain any `x`
4. independent variable include `log(x)` while data does contain `x`